packages/api-server/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  db:
+    image: postgres:13-alpine
+    environment:
+      - POSTGRES_DB=albion
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
This is not a huge deal at all since we import them anyways, but standard on this imo is PG typically since thats what postgres cli would auto import

Reply:
Not sure what you mean here, but feel free to tweak.